src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age ? <p className="Person__age">{`I am ${person.age}`}</p> : ''}
+    <p
+      className=/* {`Person ${person.partnerName ? */ "Person__partner" /* : 'Person__nopartner'}`} */
```

Comment:
The class name logic for the paragraph element is commented out. To dynamically apply the correct class based on whether the person has a partner, uncomment and fix the logic: `{`Person__${person.partnerName ? 'partner' : 'nopartner'}`}`.